### PR TITLE
Improve appinsight distributed trace setup guide and add deployment yamls

### PR DESCRIPTION
**Pull request by youngbupark**
# Description

Add local forwarder docker and deployment yamls.

* Pushed localforwarder docker image to daprio org : https://hub.docker.com/r/daprio/dapr-localforwarder/tags
* Updated docs

## Issue reference

https://github.com/dapr/components-contrib/issues/207

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Extended the documentation

### CODE REVIEW

File: howto/diagnose-with-tracing/azure-monitor.md [hunk 1]
```diff
@@ -1,28 +1,55 @@
...
+3. Get Application insights Intrumentation key from your application insights page
+4. Go to `Configure -> API Access`
+5. Click `Create API Key`
+6. Select all checkboxes under `Choose what this API key will allow apps to do:`
+   - Read telemetry
+   - Write annotations
+   - Authenticate SDK control channel
+7. Generate Key and get API key
 
 ### Setup the Local Forwarder
```
Comment (AaronCrawfis) [inline howto/diagnose-with-tracing/azure-monitor.md]: @youngbupark we should have a note here mentioning that the dapr local forwarder image is for testing purposes only, and we do not support it in production scenarios. We should point to the dockerfile and say teams can use this to create their own image

Comment (youngbupark) [reply howto/diagnose-with-tracing/azure-monitor.md]: I added some notes.I did not use word `testing` purpose. Instead, I used the word `preview`.  but feel free to edit the file directly if you want :)

Comment (AaronCrawfis) [reply howto/diagnose-with-tracing/azure-monitor.md]: Hmmm not sure if we should use the term `preview` if we don't have eventual plans to turn it into a supported feature. Don't want companies to take dependencies thinking we'll continue to develop it. But you're right, `testing` might also not be great.

@msfussell what are your thoughts?

**Comment by youngbupark** [comment]
cc/ @RicardoNiepel

File: howto/diagnose-with-tracing/azure-monitor.md (hunk 1 above)
Comment (amanbha) [reply howto/diagnose-with-tracing/azure-monitor.md]: I believe the term preview is used here for LocalForawrder which is from ApplciationInsights team and thus preview is used to refer to it.

**Pull request merged**